recipes/thrax/build.sh

```diff
@@ -0,0 +1,3 @@
+./configure --prefix="${PREFIX}"
+make -j"${CPU_COUNT}"
+make -j"${CPU_COUNT}" install
```

Comment:
Is there a `make check`?

Reply:
Yes, but it is the same as a bare `make`.